android/engine/src/main/java/org/smartregister/fhircore/engine/util/helper/TransformSupportServices.kt

```diff
@@ -79,6 +79,7 @@ class TransformSupportServices @Inject constructor(val simpleWorkerContext: Simp
       "Task_Output" -> Task.TaskOutputComponent()
       "Task_Restriction" -> Task.TaskRestrictionComponent()
       "AdverseEvent_SuspectEntity" -> AdverseEvent.AdverseEventSuspectEntityComponent()
+      "protocolApplied" -> Immunization.ImmunizationProtocolAppliedComponent()
```

Comment:
@dubdabasoduba I think we already have this implementation in place. Please check out [this](https://github.com/opensrp/fhircore/blob/9063d7b34df952a658ea550d4089692b0e546d3d/android/engine/src/main/java/org/smartregister/fhircore/engine/util/helper/TransformSupportServices.kt#L65)